octobot/src/server/admin.rs

```diff
@@ -344,7 +351,19 @@ impl MergeVersions {
             }
         }
 
-        self.make_resp(resp.set_versions(all_relevant_versions))
+        let version_url = if let Some(id) = all_relevant_versions.version_id {
```

Comment:
I thought that this should have been an Option::map(), but the lambda for that wasn't allowed to borrow-capture &merge_req, so I did this instead